README.md

```diff
@@ -35,6 +35,31 @@ When reporting on the combination of multiple products, we need to do a little m
 * Calculate average metrics for that single list using `Get-AverageReleaseMetrics`
 * Create an HTML report using `New-FourKeyMetricsReport`
 
+## How are they calculated?
+### Deployment Frequency
```

Comment:
Personal pet peeve: Can we just rename this graph to "Deployment Interval" since that's what the graph is actually showing?

Reply:
(happy to make a PR if necessary)

Reply:
I agree with your pet peeve @nyctef, but I think it's important to stick with the "industry standard" terms used in Accelerate, rather than actually being correct.